runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -474,6 +481,25 @@ String parseLevelUpWidget(WidgetInfo levelUpLevel)
 		return skillName + "(" + skillLevel + ")";
 	}
 
+	/**
+	 * Receives a String and parses it into a shortened string for filename usage.
+	 * @param killMessage killMessage String received after killing a boss,
+	 *                     with the format "Your (Boss Name) kill count is: (killcount)."
+	 * @return Shortened string in the format "(Boss Name) kill (killcount)"
+	 */
+	private String parseBossKill(String killMessage)
+	{
+		Matcher m = BOSSKILL_MESSAGE_PATTERN.matcher(killMessage);
```

Comment:
You should reuse this created matcher (what you created earlier to check if message matches_